homeassistant/components/google_assistant_sdk/notify.py

```diff
@@ -4,10 +4,32 @@
 from typing import Any
 
 from homeassistant.components.notify import ATTR_TARGET, BaseNotificationService
+from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
 
-from .helpers import async_send_text_commands
+from .const import CONF_LANGUAGE_CODE, DOMAIN
+from .helpers import async_send_text_commands, default_language_code
+
+# https://support.google.com/assistant/answer/9071582?hl=en
+LANG_TO_BROADCAST_COMMAND = {
```

Comment:
I just had one thought: Translations for UI strings typically go in the json files, and was curious if it makes sense to use that for these strings too. I can also allow users to submit more translations via lokalize and allow this to get extended to more -- however i haven't seen that done before so don't know if that causes more problems.  I could also see it's a security problem or something :)

Reply:
These aren't UI strings though. We can't let anyone enter whatever they want here in whatever language. The Google Assistant API supports a limited set of languages and the broadcast command is predefined. See the help center link at line 14 where you can replace hl to a supported language to get the broadcast command Google Assistant understands.

To extend this integration to support more languages, whenever Google Assitant SDK supports new ones, we will have to update the SUPPORTED_LANGUAGES set and the 2 failing tests will point you to a couple of dictionaries that also need to be updated. This requires code changes that we can't safely perform via localize.